io/www/src/fragment-client.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Fragment Client Library
+ * A reusable client-side library for working with content fragments
+ */
+
+// Import the modules
+import { fetchFragment } from './fragment/fetch.js';
+import { replace } from './fragment/replace.js';
+import { settings } from './fragment/settings.js';
+import { logError } from './fragment/common.js';
+
+async function previewFragment(id, options) {
+    const {
+        surface,
```

Comment:
just realised that we could have derived surface from a fragment path, no? is there a need to pass it as a param?

Reply:
this is _just_ for dictionaryId optimization for now, see line 28, pipeline does indeed detects surface later on

Reply:
so we can't reuse that detection? and skip this param?